scss/components/_button.scss

```diff
@@ -91,6 +95,9 @@ $button-responsive-expanded: false !default;
   vertical-align: middle;
   margin: $button-margin;
   font-family: $button-font-family;
+  @if ($button-font-weight != inherit && $button-font-weight != $global-weight-normal) {
+    font-weight: $button-font-weight;
+  }
```

Comment:
One could want to enforce the `inherit` value on button font-weight (because it was previously set otherwise, by an other package for example). Same for $global-weight-normal.

For all default value that should not generate any code, we should use `null`. Any property with `null` as value will not be ignored in the CSS compilation.

So you can simply set `button-font-weight` to `null`, with maybe a comment like `If different than null, set the button font-weight to the given value. Otherwise, no property is set and the browser will inherit the property.`